tfe/data_source_workspace.go

```diff
@@ -100,12 +111,19 @@ func dataSourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("queue_all_runs", workspace.QueueAllRuns)
 	d.Set("terraform_version", workspace.TerraformVersion)
 	d.Set("working_directory", workspace.WorkingDirectory)
+	d.Set("file_triggers_enabled", workspace.FileTriggersEnabled)
 	d.Set("external_id", workspace.ID)
 
 	if workspace.SSHKey != nil {
 		d.Set("ssh_key_id", workspace.SSHKey.ID)
 	}
 
+	var triggerPrefixes []string
+	if workspace.TriggerPrefixes != nil {
+		triggerPrefixes = append(triggerPrefixes, workspace.TriggerPrefixes...)
+	}
+	d.Set("trigger_prefixes", triggerPrefixes)
```

Comment:
I think you should be able to do this instead:

```suggestion
	d.Set("trigger_prefixes", workspace.TriggerPrefixes)
```